src/settings/NoticePolicy/components/EditSections/GeneralSection/GeneralSection.js

```diff
@@ -31,6 +31,7 @@ class GeneralSection extends React.Component {
     return (
       <div data-test-notice-policy-form-general-section>
         <Accordion
+          data-testid="generalTestId"
```

Comment:
as discussed in previous MR let us not use `TestId` postfix

Reply:
done